assets/constants.ts

```diff
@@ -18,8 +18,11 @@
  */
 
 export default {
-  // Whether to allow custom event reporting to Google Analytics.
-  "CAN_SEND_ANALYTICS_EVENTS": false,
+  // Analytics sandbox mode allows using mock gtag function instead of calling
+  // actual gtag library.
+  // This flag is expected to be true unless app actually requires to send
+  // analytics.
+  "ANALYTICS_SANDBOX_MODE": true,
```

Comment:
What exactly is the definition of ANALYTICS_SANDBOX_MODE? Do events get sent anywhere (even locally) or do they not get sent at all?

If the latter, then I think keeping the previous name CAN_SEND_ANALYTICS_EVENTS but changing its meaning and usage might be the way to go. It would be false in dev mode and we would make it true in prod mode. Otherwise the name change raises questions about "what exactly is sandbox mode".

Reply:
changed back to CAN_SEND_ANALYTICS_EVENTS. Added a comment.